Block/BlockContextManager.php

```diff
@@ -54,14 +53,6 @@ class BlockContextManager implements BlockContextManagerInterface
     protected $logger;
 
     /**
-     * Used for deprecation check on {@link resolve} method.
-     * To be removed in 3.0 with BC system.
```

Comment:
This comment is very old, because we are now talking about the 4.0 version. Why do we need a BC system when talking about a major change?